### ISSUE TITLE
Windows Binary not executing successfully - open config/default.yaml: file does not exist [Bug]

### ISSUE THREAD

**Issue by Raazor** [kyverno/chainsaw#1140]
### chainsaw version Version

v0.1.9

### Description

For the Windows executable it seems like the binary cannot retrieve internal paths of necessary files. 
When starting chainsaw as proposed by the quick start guide the following error message pops up:
Loading default configuration...
Error: open config\default.yaml: file does not exist

As a next step I tried creating a default config file in the folder, which leads to the next file not being found with the following error:
Error: failed to load configuration (failed to read crds\chainsaw.kyverno.io_configurations.yaml: open crds\chainsaw.kyverno.io_configurations.yaml: file does not exist)

I have tried it also on Mac and Ubuntu, which works fine, so I figure it is a Windows issue. Has somebody resolved this ?

### Steps to reproduce

1. Install binary via Go Installer or Binary download as described in the Install documentation of chainsaw
2. Run the example quick start guide
3. Run into errors

### Expected behavior

1. Default config error is shown
2. After creating your own default config "open crds\... file does not exist" error is shown. 

### Logs

```Shell
Loading default configuration...
Error: open config\default.yaml: file does not exist

Loading config (custom.config.yaml)...
Error: failed to load configuration (failed to read crds\chainsaw.kyverno.io_configurations.yaml: open crds\chainsaw.kyverno.io_configurations.yaml: file does not exist)
```

### Troubleshooting

- [X] I have searched other issues in this repository and mine is not recorded.

**Comment by eddycharly** [comment]
Thanks for reporting!

**Comment by chipzoller** [comment]
Confirmed here:

```
 .\chainsaw.exe test     
Version: 0.1.9
Loading default configuration...
Error: open config\default.yaml: file does not exist
```

**Comment by eddycharly** [comment]
Hmmm, i have an idea 🤔

**Comment by eddycharly** [comment]
Because https://github.com/golang/go/issues/44305 - fixing

**Comment by Raazor** [comment]
Thanks for the quick feedback, awesome work ! :-)

I have tried building a Windows binary and I am still getting an (other) error though:

 Loading default configuration...                                                                                                          
Error: failed to read crds\chainsaw.kyverno.io_configurations.yaml: open crds\chainsaw.kyverno.io_configurations.yaml: file does not exist

Maybe the changes were not applied on the code responsible for opening these CRDs ? 
(Sorry for not being more of help, I am not too much into Go)

**Comment by eddycharly** [comment]
Thanks for reporting @Raazor , this is the same issue but in another package, i will push a fix.

**Comment by Raazor** [comment]
Cool ! 💯

**Comment by eddycharly** [comment]
@Raazor i'm releasing v0.2.0-alpha.1 - it should fix the issue in the affected dependency, i would appreciate if you can check and eventually confirm it works now 🤞 🙏

**Comment by Raazor** [comment]
@eddycharly Can confirm it is working 🥳. Awesome ! 
On a side note: Due to the replace directory however there is currently a problem with using go install from main / v0.2.0-alpha.1. I had to download the binary from release assets to get it working. 
The error is described here: https://github.com/golang/go/issues/44840. (I did not open a bug ticket not since it was not a released version where it occurred.)
![image](https://github.com/kyverno/chainsaw/assets/9135461/d23403fd-da60-46a1-940b-52f1d5cb0d5a)
